baseline.md

```diff
@@ -1158,9 +1066,9 @@ Not yet a client of Simeon? [Get started here](https://www.simeoncloud.com/).
 
 ###### hidden-header
 
-### Baseline - Self Deploying Profile
+### Baseline%3A Self Deploying Profile
```

Comment:
the Build-BaselineMarkdown script needs to be enhanced to check for a $name property and use it if present (rather than always using the file name)